### DISCUSSION TITLE
Questions sur la saisie des champs Application

### DISCUSSION THREAD

**Discussion by adrienloyat** [dbarzin/mercator#719] [Q&A]
Bonjour
Nous commençons à explorer ce logiciel en remplacement d'un fichier excel. Merci pour cette belle réalisation.
J'ai des questions sur certaines notions de base de l'objet Application et les bonnes pratiques sur le sujet.

Je trouve le document de l'ANSSI assez flou sur certaines définitions. Qu'est-ce qui est sous-entendu par "entité responsable de l'exploitation" ? 

Il y a un champ éditeur. Est-il possible de lister l'ensemble des applications par éditeur ? 
Il y a un rapport  "Liste des entités du système d'information et de leurs applications supportées" qui semble intéressant. Est-ce qu'alors il ne convient pas de mettre les éditeurs dans les entités pour bénéficier de ce rapport ?

Notion de Responsable : est-ce qu'on y met la personne en charge du suivi applicatif à la DSI ? ou la direction à qui administre le logiciel ?
Est-il possible de sortir un tableau des responsables avec leurs applications ?

Merci d'avance d'éclairer un néophyte sur ces notions.

**Comment by dbarzin** [comment]
L’"entité responsable de l'exploitation", désigne le groupe de personnes responsable de la planification, de la mise en œuvre et de la maintenance de l’application, en veillant à ce que celle-ci réponde aux besoins des utilisateurs et des métiers de l’organisation. En d'autres termes, il s'assure que l’application fonctionne correctement, est alignée sur les objectifs stratégiques de l’organisation et respectent les délais et les coûts. 

Le champ éditeur va être ajouté à la "Liste des applications par groupe applicatif". Merci pour la remarque.

Le terme “responsable de l’application” fait référence à la personne ou à l’équipe chargée de gérer et de superviser le développement, la maintenance et l’utilisation d’une application. Le responsable de l’application peut être un développeur, un chef de projet, un administrateur système ou toute autre personne impliquée dans le cycle de vie de l’application.

Dans la liste des applications par groupe applicatif, il y a une colonne « responsable ». En triant sur cette colonne, tu obtiens les responsables avec leurs applications.

**Reply by adrienloyat** [reply]
Merci beaucoup. Ou trouvez-vous ces définitions ? Avez-vous des références (livre ou pdf) à proposer ?

**Comment by dbarzin** [comment]
C'est disponible.
